css/main.css

```diff
@@ -1,9 +1,127 @@
 
-
 /* ==========================================================================
    Author's custom styles
    ========================================================================== */
-
+   .fa {
+    padding: 20px;
+    font-size: 30px;
+    width: 50px;
+    text-align: center;
+    text-decoration: none;
+}
+
+/* Add a hover effect if you want */
+.fa:hover {
+    opacity: 0.7;
+}
+
+
+/* Facebook */
+.fa-facebook {
+    background: #3B5998;
+    color: white;
+}
+
+/* Twitter */
+.fa-twitter {
+    background: #55ACEE;
+    color: white;
+}
+
+.fa-google {
+  background: #dd4b39;
+  color: white;
+}
+
```

Comment:
I agree with other teammates that we should limit the number of icons to five. Everything below Google can be removed, with the exception of Pinterest and Insta. For images and poetry, many of these don't apply and wouldn't make sense to have (e.g., Spotify).